### PR TITLE
I803 complete milestone

### PR DESCRIPTION
**Pull request by Serhiy91**

### CODE REVIEW

File: app/imports/ui/react/goals/components/ChartActions.js [hunk 1]
```diff
@@ -13,9 +13,14 @@ const StyledListGroup = styled(ListGroup)`
...
-const ChartActions = ({ onEdit, onDelete, onComplete }) => (
+const ChartActions = ({
+  onEdit,
+  onDelete,
+  onComplete,
+  isCompleteBtn,
+}) => (
   <StyledListGroup>
-    {onComplete && (
+    {isCompleteBtn && onComplete && (
```
Comment (Dartv) [inline app/imports/ui/react/goals/components/ChartActions.js]: Why add this prop? You can conditionally pass `onComplete` in parent component using `branch`

File: app/imports/ui/react/goals/containers/MilestoneChartActionsContainer.js [hunk 2]
```diff
@@ -12,6 +12,9 @@ const enhance = compose(
   withStore,
   connectUI(),
   onlyUpdateForKeys(['_id', 'title', 'goalId']),
+  withProps(({ isCompleted }) => ({
```
Comment (Dartv) [inline app/imports/ui/react/goals/containers/MilestoneChartActionsContainer.js]: use `branch` here with handler

**Comment by Serhiy91** [comment]
@Dartv fixed

File: app/imports/ui/react/goals/containers/GoalsChartActionsContainer.js [hunk 3]
```diff
@@ -9,13 +9,14 @@ import ChartActions from '../components/ChartActions';
 const enhance = compose(
   withStore,
   connectUI(),
+  flattenProp('goal'),
```
Comment (Dartv) [inline app/imports/ui/react/goals/containers/GoalsChartActionsContainer.js]: You can use local `onlyUpdateForKeysDeep` in this case.

**Pull request merged**